internal/cli/atlas/kubernetes/config/apply.go

```diff
@@ -93,7 +93,7 @@ func (opts *ApplyOpts) Run() error {
 		return err
 	}
 
-	featureValidator, err := features.NewAtlasCRDs(opts.crdsProvider, opts.operatorVersion)
+	atlasCRDs, err := features.NewAtlasCRDs(opts.crdsProvider, opts.operatorVersion)
```

Comment:
Not sure about renaming this, but ok

Reply:
this serves two functions now:
1. feature validation
2. patching

Reply:
generally: the fact that this struct implements an interface is ok but it might implement more. naming it after the interface is confusing, now it would have to be called `featureValidatorAndPatcher`.